lib/travis/build/script/julia.rb

```diff
@@ -91,19 +91,21 @@ def julia_url
             case config[:os]
             when 'linux'
               osarch = 'linux/x64'
+              nightlyosarch = osarch
               ext = 'linux-x86_64.tar.gz'
               nightlyext = 'linux64.tar.gz'
             when 'osx'
               osarch = 'osx/x64'
+              nightlyosarch = 'mac/x64'
```

Comment:
Instead of doing this, we should just copy the `osx` directory to `mac` on the release side of things and keep the `osx` folder around for a while until we get tired of it and drop the old links.  But we should strive for consistency when we can.

Reply:
I agree, this will make it much easier for tooling to support this change without requiring logic changes.

Reply:
Done.